### PR TITLE
Consolidate all db maintenance of pruner to operator

### PR DESCRIPTION
**Pull request by cdlliuy**
[#158935223]

### CODE REVIEW

File: src/integration/integration_operator_others.go [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+				})
+			})
+
+			Context("when update a policy to another schedule sets only in policy DB without any update in scheduler ", func() {
+				BeforeEach(func() {
+					doAttachPolicy(testAppId, []byte(policyStr), http.StatusCreated, INTERNAL)
+					Expect(checkSchedule(testAppId, http.StatusOK, map[string]int{"recurring_schedule": 4, "specific_date": 2})).To(BeTrue())
+
+					By("update policy in policy db directly")
```
Comment (boyang9527) [inline src/integration/integration_operator_others.go]: Remove "by" given there is no assertion in this section.

File: src/integration/integration_operator_others.go [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+					newPolicyStr := string(readPolicyFromFile("fakePolicyWithScheduleAnother.json"))
+					deletePolicy(testAppId)
+					insertPolicy(testAppId, newPolicyStr, testGuid)
+
+					By("the schedules should not be updated before operator triggers the sync")
```
Comment (boyang9527) [inline src/integration/integration_operator_others.go]: remove this "By" too

File: src/integration/integration_operator_others.go [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+				ScalingType:  models.ScalingTypeDynamic,
+				Status:       models.ScalingStatusSucceeded,
+				Error:        "",
+			}
+			insertScalingHistory(history)
+			Expect(getScalingHistoryTotalCount(testAppId)).To(Equal(1))
+
+		})
+
+		It("operator should sync the updated schedule to scheduler ", func() {
```
Comment (boyang9527) [inline src/integration/integration_operator_others.go]: this is not correct, this is about prune the database, not sync schedule

File: src/integration/integration_suite_test.go [hunk 4]
```diff
@@ -265,6 +274,10 @@ func stopEventGenerator() {
 func stopServiceBroker() {
 	ginkgomon.Kill(processMap[ServiceBroker], 5*time.Second)
 }
+func stopOperator() {
+	ginkgomon.Interrupt(processMap[Operator], 5*time.Second)
```
Comment (boyang9527) [inline src/integration/integration_suite_test.go]: why use Interrupt while kill for others

**Comment by qibobo** [comment]
LGTM

**Pull request merged**